sql/synth_recipes.sql

```diff
@@ -4018,6 +4017,56 @@ INSERT INTO `synth_recipes` VALUES (3906,1,0,0,114,0,0,0,0,70,0,4099,4241,1394,8
+INSERT INTO `synth_recipes` VALUES (3926,1,0,0,0,0,63,0,0,0,0,4101,4243,5135,5449,5454,0,0,0,0,0,17904,17904,17904,17904,4,6,8,10); -- Briny Broth
+INSERT INTO `synth_recipes` VALUES (3927,1,0,0,0,0,0,0,0,0,39,4103,4245,17868,17868,0,0,0,0,0,0,17894,17894,17894,17894,4,6,8,10); -- Vermihumus
+INSERT INTO `synth_recipes` VALUES (3928,1,0,0,0,0,0,0,0,0,39,4103,4245,533,533,0,0,0,0,0,0,17894,17894,17894,17894,4,6,8,10); -- Vermihumus
+INSERT INTO `synth_recipes` VALUES (3929,1,0,0,0,0,0,0,0,0,39,4103,4245,17869,17869,0,0,0,0,0,0,17894,17894,17894,17894,6,8,10,12); -- Vermihumus
+INSERT INTO `synth_recipes` VALUES (3930,1,0,0,0,0,45,0,0,0,0,4098,4240,2155,2155,17397,0,0,0,0,0,17893,17893,17893,17893,4,6,8,10); -- Wool Grease
+INSERT INTO `synth_recipes` VALUES (3931,1,0,0,0,0,0,0,0,0,66,4098,4240,2761,0,0,0,0,0,0,0,2762,2762,2762,2762,1,2,3,4); -- Feyweald Lumber
+INSERT INTO `synth_recipes` VALUES (3932,1,0,0,0,0,0,100,0,0,0,4096,4238,4023,4023,4023,4023,0,0,0,0,4024,4024,4024,4024,1,1,1,1); -- Snowsteel
+INSERT INTO `synth_recipes` VALUES (3933,1,0,0,0,0,100,0,0,0,0,4096,4238,936,4375,4387,4449,4509,5680,0,0,5930,5931,5931,5931,1,1,1,1); -- Sprightly Soup
+INSERT INTO `synth_recipes` VALUES (3934,1,0,0,0,0,0,78,0,0,0,4098,4240,679,0,0,0,0,0,0,0,2651,2651,2651,2651,6,8,10,12); -- Fusion Bolt Heads
+INSERT INTO `synth_recipes` VALUES (3935,1,0,0,0,0,0,0,0,0,65,4099,4241,1243,3927,0,0,0,0,0,0,21323,21323,21323,21323,33,66,99,99); -- Oxidant bolt
+INSERT INTO `synth_recipes` VALUES (3936,1,1987,0,0,0,0,0,0,0,65,4099,4241,1243,1243,1243,1657,3927,3927,3927,0,21323,21323,21323,21323,99,99,99,99); -- Oxidant bolt - KI Boltmaker
+INSERT INTO `synth_recipes` VALUES (3937,1,0,0,0,0,102,0,0,0,0,4103,4245,95,583,612,614,936,1891,4273,0,5909,5909,5909,5909,1,1,1,1); -- Pickled Rarab Tail
+INSERT INTO `synth_recipes` VALUES (3938,1,0,0,0,0,103,0,0,0,0,4096,4238,615,627,2237,4509,5568,5661,0,0,5922,5923,5923,5923,33,33,66,99); -- Walnut Cookie
+INSERT INTO `synth_recipes` VALUES (3939,1,0,0,0,0,104,0,0,0,0,4101,4243,5534,5536,5963,0,0,0,0,0,21492,21493,21493,21493,6,4,8,12); -- Insipid Broth
+INSERT INTO `synth_recipes` VALUES (3940,1,0,0,0,0,104,0,0,0,0,4103,4245,703,2016,4031,4509,0,0,0,0,21494,21495,21495,21495,4,2,4,8); -- Wetlands Broth
+INSERT INTO `synth_recipes` VALUES (3941,1,0,0,0,0,104,0,0,0,0,4096,4238,898,898,4387,4435,4509,5234,9194,0,9197,9197,9197,9197,6,6,10,10); -- Soy Ramen Soup
+INSERT INTO `synth_recipes` VALUES (3942,1,0,0,0,0,104,0,0,0,0,4096,4238,614,617,898,898,4509,5234,9194,0,9198,9198,9198,9198,6,6,8,10); -- Miso Ramen Soup
+INSERT INTO `synth_recipes` VALUES (3943,1,0,0,0,0,104,0,0,0,0,4096,4238,936,1019,4379,4399,4461,4509,5131,0,9199,9199,9199,9199,6,6,6,6); -- Salt Ramen Soup
+INSERT INTO `synth_recipes` VALUES (3944,1,0,0,0,0,105,0,0,0,0,4096,4238,627,2156,4509,5566,0,0,0,0,5926,5927,5927,5927,1,1,1,1); -- Date Tea
```

Comment:
thats what happens when you don't control+f in a file that you know is 99% not commented 😛